samples/CognitiveSearch/Generated/IndexersRestClient.cs

```diff
@@ -595,7 +595,7 @@ public async Task<Response<IndexerExecutionInfo>> GetStatusAsync(string indexerN
         /// <param name="requestOptions"> Parameter group. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="indexerName"/> is null. </exception>
-        public Response<IndexerExecutionInfo> GetStatus(string indexerName, RequestOptions requestOptions = null, CancellationToken cancellationToken = default)
+        public Response<IndexerExecutionInfo> GetStatus(string indexerName, Models.RequestOptions requestOptions = null, CancellationToken cancellationToken = default)
```

Comment:
Why `Models` is added here rather than in `using` namespace?

Reply:
We added a new [RequestOptions](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/RequestOptions.cs) type in Azure.Core and I think it added `Models.` to resolve the conflict(I didn't do any change related to this).

Reply:
ah yes that would explain it.  @AlexanderSher are we good to go here?

Reply:
The RequestOptions in Azure.Core is in the Azure namespace, e.g. `Azure.RequestOptions`.  I'm not sure what the one in the Models namespace is, @ShivangiReja, do you know?

Reply:
Responded here - https://github.com/Azure/autorest.csharp/pull/2025#discussion_r818080997

Reply:
Ah, I see!  There is another `RequestOptions` coming from Search, and it's not the DPG one!